usaspending_api/references/management/commands/load_naics.py

```diff
@@ -73,5 +92,5 @@ def load_naics(path, append):
         wb = load_workbook(filename=path)
```

Comment:
This sorts the files in the wrong order because the names are not uniform:

- naics2007.xlsx
- 2002 naics.xlsx
- 2-digit_2012_Codes.xlsx
- 2-6 digit_2017_Codes.xlsx

Need to either rename the files or come up with a more clever sorting algorithm.

Reply:
> This sorts the files in the wrong order because the names are not uniform:
> 
>     * naics2007.xlsx
> 
>     * 2002 naics.xlsx
> 
>     * 2-digit_2012_Codes.xlsx
> 
>     * 2-6 digit_2017_Codes.xlsx
> 
> 
> Need to either rename the files or come up with a more clever sorting algorithm so the files are loaded in chronological order.

Hmm.   After looking through the code s'more, I see there's a check later to not update newer values with older stuff.  Perhaps this is ok.  Still, though, those file names....

Reply:
I was a little miffed when I first looked at the code to just parse the year of regex, but I do kind of like that this preserves the names of the files as they come from the website (kinda). That said, upon re-reading the ticket the technical details do dictate the filenames, so I'll change them around.